### DISCUSSION TITLE
Changed by another program

### DISCUSSION THREAD

**Discussion by Bayes2** [freeplane/freeplane#484] [Help]
I get the message "Mind map file was changed by another program. Reload?" all the time. And when I say No nothing happens and if I say Yes my changes are lost. 
What is causing this?
(Should I have posted this under 'issues'?)

**Comment by bepolymathe** [comment]
Do you use other programs? Do you use a synchronization solution between several machines?

**Reply by Bayes2** [reply]
I dont use other Mindmap-programs. But I use other programs as Word, R, Excel etc. 

I work a lot connected to remote servers through VPN.  

I have several maps and I only get this message to one of them.

**Reply by Illegit** [reply]
in my case this happens with notepad++
illegit

**Reply by vladimir-from-adizes** [reply]
For me "Mind map file was changed by another program. Reload?" happens after I save map which is in the folder controlled by **GoogleDrive** software. Little annoying - I just saved it, why reload. Is this some weird GoogleDrive behavior or FreePlane monitors more file manipulations than it should? Probably GoogleDrive changes some property that file is synced, that shouldn't be considered as changed file.

**Comment by ldmpub** [comment]
Hi,

I confirm: I have noticed, in the past, the same behavior on my side with Freeplane + Google Drive + Windows but this has vanished with recent Freeplane version ; for instance I can not reproduce it with FP 1.10.3.

What version of Freeplane do you run?

Notes: 
* my first workaround was to work with a "local copy" of the mindmap file then, when "my Freeplane work" is finished, to copy it to my "shared Google Drive"
* then I used an other sync tool (NextCloud) where this "reload" problem with popups never occurred
* now this "wrong reload" has disappeared since, at least, the 2 last FreePlane versions, even with Google Drive as sync files tool ...

**Reply by Bayes2** [reply]
I use 1.10.3

**Comment by halfbeing** [comment]
I'm not at the same computer now, but the version is whatever the stable version for Mac was yesterday when I downloaded it, which I believe is 1.10.3. 

The mind map was indeed saved to my iDrive synced folder. I have many types of file in that folder, but it is only with Freeplane that I get this message.

**Comment by dpolivaev** [comment]
You can disable option "Preferences…->Environment->Load->Monitor external mind map file changes"

**Reply by halfbeing** [reply]
As I said in my OP (which seems to have disappeared when it was merged with another question), I will sometimes edit the same mind map from different machines, so I would actually like to have a warning when the file genuinely has changed. It is only the false warnings that I want to stop.

**Comment by Tibi** [comment]
This happens because your Google Drive is configured to not store files locally (mirror) but to only in the cloud (stream).
To fix it, in the Google Drive preferences, choose Mirror files instead of Stream files.
